### ISSUE TITLE
error running fad stats

### ISSUE THREAD

**Issue by rhalperin** [blachlylab/fade#25]
I am getting the following error running fade stats:
```
rt/dwarfeh.d:354: uncaught exception reached top of stack
This might happen if you're missing a top level catch in your fiber or signal handler
core.exception.OutOfMemoryError@core/lifetime.d(137): Memory allocation failed
----------------
src/env/__libc_start_main.c:10 <ERROR: Unable to retrieve function name> [0x0]
Aborted
```
Here is how I am running fade:
```
docker run -it -m 8g -v $PWD:/data --entrypoint sh blachlylab/fade
fade stats $BAM
```
My host machine is a Mac.  I've also been getting the same error on a cloud workstation running ubuntu 18.04 with fade installed via conda.  I've tried several different bams including one that was trivially sized (0.5G) and have checked that I have plenty of free memory available on both machines.  I am able to run `fade out` on the same bams without any errors.

**Comment by charlesgregory** [comment]
This error is misleading from the `D` language, `fade stats $BAM` should be run as `fade stats $BAM output.tsv`. Though there are other issues I found when investigating this so I will push up a new version in which `fade stats $BAM` works by writing to `stdout`.

**Comment by charlesgregory** [comment]
Can you try [v0.5.6](https://github.com/blachlylab/fade/releases/tag/v0.5.6)?

**Comment by rhalperin** [comment]
Running it as `fade stats $BAM output.tsv` worked.  Thanks!

If you get the docker image and/or the conda installation updated to 0.5.6, then I will give it a try.

**Comment by charlesgregory** [comment]
The new docker image was pushed up as well with the release, you may have to run `docker pull blachlylab/fade` to get the latest version. conda will automatically update but it take probably a day or two for it to get worked into bioconda. If you get the same error, then there could be an issue with the docker image update process I use.

**Comment by rhalperin** [comment]
I pulled the latest docker, and I do see fade v0.5.6.  However, I am getting a different error now:
``` 
fade stats $BAM
[E::hts_open_format] Failed to open file "stats" : No such file or directory
Segmentation fault
```

**Comment by charlesgregory** [comment]
Can you try `v0.5.7`? I found out in fixing this issue I introduced another which should now be fixed.

**Comment by rhalperin** [comment]
Yes, it appears fixed in 0.5.7, thanks!
